build.gradle

```diff
@@ -155,4 +157,21 @@ configure(coreProjects) {
         }
     }
     check.dependsOn jacocoTestCoverageVerification
-}
\ No newline at end of file
+}
+
+licenseReport {
+    excludeOwnGroup = true
+    excludeBoms = false
+    renderers = new TextReportRenderer()
+
+}
+
+task generateThirdPartyReport(type: Copy) {
+    from 'build/reports/dependency-license/'
+    into '.'
+    include '*.txt'
```

Comment:
I think it would be better to only copy the file that is needed. Otherwise, if the plugin changes to output multiple files they will end up in the root. This could end up being confusing.

Can this line just be?
```
include 'THIRD-PARTY-NOTICES.txt'
```

Reply:
Yes